### DISCUSSION TITLE
Windows Terminal v1.12.10393.0

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#12476] [Announcements]
This is a quick servicing release to fix a couple of big issues in the initial 1.12 stable drop.

This release was made available to insiders in the Dev and Beta channels on February 10, and will be generally available to everyone shortly afterwards. You can always install the update from this release page or using `winget`.

<details>
<summary>Preinstallation Kit info</summary>

A [preinstallation](https://docs.microsoft.com/en-us/windows/msix/desktop/deploy-preinstalled-apps) kit is available for system integrators and OEMs interested in prepackaging Windows Terminal with a Windows image. More information is available in the [DISM documentation on preinstallation](https://docs.microsoft.com/windows-hardware/manufacture/desktop/preinstall-apps-using-dism). Users who do not intend to preinstall Windows Terminal should continue using the _msixbundle_ distribution.

</details>

## Bug Fixes

* Terminal is once again localized (#12375)
   * We encountered an issue in automatic language detection, which resulted in our package's languages
     being ignored.
* "Default Terminal" profile matching now works better for profiles containing unquoted whitespace (#12348)
* We believe we have fixed an unusual crash on launch in constructing the taskbar Jump List (#12430)
* Symbols are now published to the public symbol server! Woot! (#12441)
* We've fixed a crash that used to surface while Magnifier or other assistive technologies were running (#12436)
* Ubuntu users with overridden commandlines will no longer see a confusing "`~` could not be found" error (#12437)

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.12.10393.0'>Windows Terminal v1.12.10393.0</a>.</em>

**Comment by bzzrak** [comment]
I'm not seeing 1.12 in the Store at all, still on 1.11.3471.0. Is that normal? I'm using the stable terminal, not the preview.

**Reply by aisgbnok** [reply]
[I think they have started using a "slow rollout" system so you may not receive the latest build immediately.](https://github.com/microsoft/terminal/releases/tag/v1.9.1942.0#:~:text=trying%20a%20new-,%22slow%20rollout%22,-system%20that%20follows) Coupled with the localization issues in `v1.12.10334.0` they may have pulled it temporarily. You will probably receive this fixed `v1.12.10393.0` sometime next week.

**Reply by WizardCM** [reply]
Still not on the Store it seems.

**Reply by aisgbnok** [reply]
Yeah, it seems like a really slow rollout this time. If your eager to update, [you can download the `.msixbundle` from the release.](https://github.com/microsoft/terminal/releases/download/v1.12.10393.0/Microsoft.WindowsTerminal_1.12.10393.0_8wekyb3d8bbwe.msixbundle) Running the bundle will trigger windows app installer to update Terminal using your bundle.

**Reply by mcswell** [reply]
And when I click on "install" at the store, it just spins.  I think the Microsoft store is just broken, glad I can install it from github.

**Reply by krystofkrticka** [reply]
I also have this problem it's almost a month from release but I still didn't get this update using Microsoft Store. Well I think I'll start updating Windows Terminal using `winget` instead of Microsoft Store

**Reply by zadjii-msft** [reply]
(more details in: https://github.com/microsoft/terminal/discussions/12346#discussioncomment-2275648)

**Comment by afkvido** [comment]
cool

**Comment by stellarfeline** [comment]
Cannot install in MS Store. 
Code: 0x80073D0D

**Reply by DHowett** [reply]
<html>
<body>


ERROR_INSTALL_VOLUME_OFFLINE | The deployment operation failed because the volume is offline. For a package update, the volume refers to the installed volume of all package versions.
-- | --



</body>
</html>


Do you have Windows set to install "store" applications to another drive?

**Reply by stellarfeline** [reply]
Yes, I just changed the installation drive.

I solved this problem by uninstalling the app in settings first.
